### PR TITLE
Drawer added

### LINKED ISSUE #18: implement burgermenu/settings UI
( will contain: add contact/contact list/sign out )

### PR DESCRIPTION
**Pull request by SauleStan**
drawer

### CODE REVIEW

File: lib/screens/home_screen.dart [hunk 1]
```diff
@@ -11,14 +11,14 @@ class HomeScreen extends StatelessWidget {
       appBar: AppBar(
         elevation: 8,
         // Menu button
-        leading: IconButton(
-          icon: Icon(Icons.menu),
-          onPressed: () {}, //TODO: Implement menu functionality
-        ),
+        // leading: IconButton(
```
Comment (crowjambo) [inline lib/screens/home_screen.dart:14]: dont leave comments in production

File: lib/screens/home_screen.dart [hunk 2]
```diff
@@ -11,14 +11,14 @@ class HomeScreen extends StatelessWidget {
...
+        //   icon: Icon(Icons.menu),
+        //   onPressed: () {}, //TODO: Implement menu functionality
+        // ),
         title: Text('Inbox'),
       ),
       body: ListView.builder(
-          // I'm honestly not sure what this is for. TODO: figure it out
+          // I'm honestly know what this is for.
```
Comment (crowjambo) [inline lib/screens/home_screen.dart:21]: dont leave comments in production

File: lib/screens/home_screen.dart [hunk 3]
```diff
@@ -159,6 +159,37 @@ class HomeScreen extends StatelessWidget {
               ),
             );
           }),
+        drawer: Drawer(
+                child: ListView(
+                  // Important: Remove any padding from the ListView.
```
Comment (crowjambo) [inline lib/screens/home_screen.dart:164]: if its something that needs to be done, create a ToDo task in project board for it

**Review by crowjambo — CHANGES_REQUESTED** [review]

File: lib/screens/home_screen.dart (hunk 1 above)
Comment (SauleStan) [reply lib/screens/home_screen.dart:14]: ):

**Pull request merged**